lib/rules/no-negated-condition.js

```diff
@@ -59,15 +63,30 @@ export default class NoNegatedCondition extends Rule {
       const isIfElseBlockStatement = node.type === 'BlockStatement' && node.inverse;
       const isIfElseNonBlockStatement = node.type !== 'BlockStatement' && node.params.length === 3;
       const shouldFlipCondition = isIfElseBlockStatement || isIfElseNonBlockStatement;
-      const message = isUnless
+      const message = doubleNegation
```

Comment:
Rename to `hasDoubleNegation` to be consistent with other boolean variable names.